doc/lua/imgui.md

```diff
@@ -27,14 +37,182 @@ The ImGui library allows plugins to render UI elements.
 
 # Enumerations
 
-## ImGuiWindowFlags
+## WindowFlags
 
-## ImGuiComboFlags
+| Name | 
+| ---- |
+| None |
+| NoTitleBar |
+| NoResize |
+| NoMove |
+| NoScrollbar |
+| NoScrollWithMouse |
+| NoCollapse |
+| AlwaysAutoResize |
+| NoBackground |
+| NoSavedSettings |
+| NoMouseInputs |
+| MenuBar |
+| HorizontalScrollbar |
+| NoFocusOnAppearing |
+| NoBringToFrontOnFocus |
+| AlwaysVerticalScrollbar |
+| AlwaysHorizontalScrollbar |
+| AlwaysUseWindowPadding |
+| NoNavInputs |
+| NoNavFocus |
+| UnsavedDocument |
+| NoDocking |
 
-## ImGuiSelectableFlags
 
-## ImGuiTableColumnFlags
+## ComboFlags
 
-## ImGuiTableFlags
+| Name |
+| ---- |
+| None |
+| PopupAlignLeft |
+| HeightSmall |
+| HeightRegular |
+| HeightLarge |
+| HeightLargest |
+| NoArrowButton |
+| NoPreview |
 
-## ImGuiButtonFlags
+## SelectableFlags
+
+| Name |
+| ---- |
+| None |
+| DontClosePopups |
+| SpanAllColumns |
+| AllowDoubleClick |
+| Disabled |
+| AllowItemOverlap |
+| NoHoldingActiveID |
+| SelectOnNav |
+| SelectOnClick |
+| SelectOnRelease |
+| SpanAvailWidth |
+| DrawHoveredWhenHeld |
+| SetNavIdOnHover |
+| NoPadWithHalfSpacing |
+
+## TableColumnFlags
+
+| Name |
+| ---- |
+| None |
+| Disabled |
+| DefaultHide |
+| DefaultSort |
+| WidthStretch |
+| WidthFixed |
+| NoResize |
+| NoReorder |
+| NoHide |
+| NoClip |
+| NoSort |
+| NoSortAscending |
+| NoSortDescending |
+| NoHeaderLabel |
+| NoHeaderWidth |
+| PreferSortAscending |
+| PreferSortDescending |
+| IndentEnable |
+| IndentDisable |
+| IsEnabled |
+| IsVisible |
+| IsSorted |
+| IsHovered |
+
+## TableFlags
+
+| Name |
+| ---- |
+| None |
+| Resizable |
+| Reorderable |
+| Hideable |
+| Sortable |
+| NoSavedSettings |
+| ContextMenuInBody |
+| RowBg |
+| BordersInnerH |
+| BordersOuterH |
+| BordersInnerV |
+| BordersOuterV |
+| BordersH |
+| BordersV |
+| BordersInner |
+| BordersOuter |
+| Borders |
+| NoBordersInBody |
+| NoBordersInBodyUntilResize |
+| SizingFixedFit |
+| SizingFixedSame |
+| SizingStretchProp |
+| SizingStretchSame |
+| NoHostExtendX |
+| NoHostExtendY |
+| NoKeepColumnsVisible |
+| PreciseWidths |
+| NoClip |
+| PadOuterX |
+| NoPadOuterX |
+| NoPadInnerX |
+| ScrollX |
+| ScrollY |
+| SortMulti |
+| SortTristate |
+
+## InputTextFlags
+
+| Name | Notes |
+| ---- ||
```

Comment:
This table doesn't render - I think it needs a hyphen under Notes.

Reply:
Ah thanks, VS is more forgiving